cli_reference/opm/cli-opm-ref.adoc

```diff
@@ -31,6 +31,10 @@ include::modules/opm-cli-ref-init.adoc[leveloffset=+1]
 include::modules/opm-cli-ref-render.adoc[leveloffset=+1]
 include::modules/opm-cli-ref-validate.adoc[leveloffset=+1]
 include::modules/opm-cli-ref-serve.adoc[leveloffset=+1]
+
+:FeatureName: The SQLite-based catalog format, including the related CLI commands,
```

Comment:
Shouldn't this be a period instead of a comma at the end of the sentence?

Ignore that question...I see in the output that there is a phrase after the comma that is not showing in the asciidocs clip.